src/java/org/apache/cassandra/db/filter/SliceQueryFilter.java

```diff
@@ -276,10 +281,22 @@ public Comparator<Cell> getColumnComparator(CellNameType comparator)
 
     public void collectReducedColumns(ColumnFamily container, Iterator<Cell> reducedColumns, DecoratedKey key, int gcBefore, long now)
     {
-        reducedCells = CountingCellIterator.wrapIterator(reducedColumns, now, gcBefore);
+        reducedCells = CountingCellIterator.wrapIterator(reducedColumns, metrics, now, gcBefore);
         columnCounter = columnCounter(container.getComparator(), now);
-        deletionInfo = container.deletionInfo();
```

Comment:
how does `deletionInfo` get set here before we've itereated through the `reducedCells`?

Reply:
discussed offline